### DISCUSSION TITLE
Add novelwriter to NixOS repo

### DISCUSSION THREAD

**Discussion by mogplus88** [vkbo/novelWriter#1650] [Ideas]
Hi,
I just started playing with NixOS and so far I am very impressed. It's package management system is radically different, in fact the whole system configuration philosophy is radically different, but I'm slowly coming to terms with it.
Unfortunately novelwriter is not in the NixOS repo, and installing a package that is not in the repo seems to be pretty challenging from the brief search I have done. Just wondering if you are considering adding it to the NixOS repo?
;-)

**Comment by vkbo** [comment]
I'm trying to keep the number of release methods low so that it's maintainable for me, focusing on one per platform. For Linux, that is AppImage. I only keep Debian packages around because I already made them, and it was a lot of work.

If other people want to maintain different release channels for novelWriter, that's fine with me.
